test/testcafe/framework/page-objects/DeviceChallengePollPageObject.js

```diff
@@ -27,6 +27,9 @@ export default class DeviceChallengePollViewPageObject extends BasePageObject {
   }
 
   getAppLinkContent() {
+    if(userVariables.v3) {
+      return this.form.getByText('If Okta Verify did not open automatically, tap Open Okta Verify.').innerText;
+    }
```

Comment:
This is a bit redundant, since the returned text will just be the text passed to `getByText`.

In these cases I like to try to update the method to be more like `hasAppLinkContent` and allow the `getByText` to throw when it's not there, or still leave it as `getAppLinkContent` but update the selection to `return queryByText('target text')` and then also update the test to assert that the element returned exists.